lms/static/sass/course/instructor/_instructor_2.scss

```diff
@@ -1979,3 +1979,56 @@ input[name="subject"] {
   left: 2em;
   right: auto;
 }
+
+.course-specific-container {
+
+  .upload {
+    position: relative;
+
+    button{
+      position: absolute;
+      top: 0;
+      left: 0;
+      z-index: 1;
+    }
+
+    .fileupload {
+      position: relative;
+      z-index: 2;
+      opacity: 0;
+      height: 30px;
+      width: 105px;
+    }
+  }
+
+  .bordered-fieldset {
+    border: solid 4px #000000;
+    padding: 20px;
+    margin-top: 30px;
+  }
+
+  .error {
+    color: #E40004;
+  }
+
+  .success {
+    color: #008000;
+  }
+
+  .new-section {
+    margin-top: 20px;
+  }
+
+  .select-tag {
+    width: 40%;
+    height: 35px;
+    font-size: 1.1em;
+  }
+
+  .output-box {
+    .loading {
+      font-size: 2em;
+      margin-bottom: 20px;
+    }
+  }
+}
```

Comment:
from `_variables.scss` it's set to:

```
// BASE
$baseline: 20px;
```

@azimpradhan - just change the scss to look like this:

```
.course-specific-container {

  .upload {
    position: relative;

    button{
      position: absolute;
      top: 0;
      left: 0;
      z-index: 1;
    }

    .fileupload {
      position: relative;
      z-index: 2;
      opacity: 0;
      height: ($baseline*1.5);
      width: ($baseline*6);
    }
  }

  .bordered-fieldset {
    border: solid 4px #000000;
    padding: $baseline;
    margin-top: ($baseline*1.5);
  }

  .error {
    color: #E40004;
  }

  .success {
    color: #008000;
  }

  .new-section {
    margin-top: $baseline;
  }

  .select-tag {
    width: 40%;
    height: ($baseline*2);
    font-size: 1.1em;
  }

  .output-box {
    .loading {
      font-size: 2em;
      margin-bottom: $baseline;
    }
  }
}
```